components/ItemCard.js

```diff
@@ -65,7 +77,7 @@ export default function ItemCard({
           <StyledQty>Qty:</StyledQty>
           <StyledValue>{quantity}</StyledValue>
           <StyledDate>date:</StyledDate>
-          <StyledDateValue>{date}</StyledDateValue>
+          <StyledDateValue>{changeDate(dateString)}</StyledDateValue>
```

Comment:
You should be fine with just passing the `date`.
```suggestion
          <StyledDateValue>{changeDate(date)}</StyledDateValue>
```